projects/portland-metro/.env

```diff
@@ -1,3 +1,3 @@
 COMPOSE_PROJECT_NAME=pelias
-DATA_DIR=/tmp/pelias/portland-metro
-DOCKER_USER=1000
+DATA_DIR=./data
```

Comment:
This change seems unrelated, could you please revert it?

Reply:
Yes is unrelated. I have reverted the commit. Please review again.